ethsign-crypto 0.1.0 from crates.io causes compile failure with ethsign 0.7.1

```
error[E0308]: mismatched types
   --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/ethsign-0.7.1/src/keyfile.rs:130:69
    |
130 |             crypto::derive_key_iterations(password.as_ref(), &salt, iterations);
    |                                                                     ^^^^^^^^^^ expected struct `std::num::NonZeroU32`, found u32
    |
    = note: expected type `std::num::NonZeroU32`
               found type `u32`

```

Cargo.toml snippet
```
ethsign = {version = "0.7.1", default-features = false, features = ["pure-rust"]}
ethsign-crypto = "0.1.0"
```
Am I doing something wrong?

It seems ethsign-crypto 0.1.0 source has a breaking change that was published to crates.io also as 0.1.0. Am I misreading something?
 
Do you need to bump the ethsign-crypto minor version and publish ethsign-crypto 0.2.0?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -7,7 +7,7 @@ homepage = "https://github.com/tomusdrw/ethsign"
 license = "GPL-3.0"
 name = "ethsign"
 repository = "https://github.com/tomusdrw/ethsign"
-version = "0.7.1"
+version = "0.7.2"
 
 [dependencies]
 zeroize = "0.10.0"
@@ -19,7 +19,7 @@ serde = { version = "1.0", features = ["derive"]}
 
 # Libraries for for pure-rust crypto
 libsecp256k1 = { package="libsecp256k1", version = "0.2.2", optional = true }
-ethsign-crypto = { version = "0.1", path = "./ethsign-crypto", optional = true }
+ethsign-crypto = { version = "0.2", path = "./ethsign-crypto", optional = true }
 
 [dev-dependencies]
 serde_json = "1.0"

--- a/ethsign-crypto/Cargo.toml
+++ b/ethsign-crypto/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "ethsign-crypto"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 documentation = "https://docs.rs/crate/ethsign-crypto"

